services/blockchain-indexer/shared/dataService/business/schemas.js

```diff
@@ -69,6 +69,8 @@ const getAllSchemas = async () => {
 			assets: assetsSchemas,
 			commands: commandsParamsSchemas,
 			events: eventsSchemas,
+			// TODO: Fetch schema from node once this issue is closed https://github.com/LiskHQ/lisk-sdk/issues/8375
+			ccm: {},
```

Comment:
I think instead of returning empty object we can return hard-coded ccmSchema until is exposed from SDK. What do you think @sameersubudhi ? 
Ref: https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/interoperability/schemas.ts#L37

Reply:
Applied.